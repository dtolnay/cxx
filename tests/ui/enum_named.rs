#[cxx::bridge]
mod ffi {
    enum Bad {
        A{age: i32},
        B(i32),
    }
}

fn main() {}
