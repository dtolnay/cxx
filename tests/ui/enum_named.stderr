error: Named variants are not supported
 --> tests/ui/enum_named.rs:3:5
  |
3 | /     enum Bad {
4 | |         A{age: i32},
5 | |         B(i32),
6 | |     }
  | |_____^
